src/api/posts/src/routes/feeds.js

```diff
@@ -34,6 +34,32 @@ feeds.get('/', async (req, res, next) => {
   );
 });
 
+//This should return a JSON array of Objects with the feed info as well as the reason why it was invalidated.
+feeds.get('/invalid', async (req, res, next) => {
+  let keys = [];
```

Comment:
I would rename this `invalidFeeds` vs. `keys`. Also, no need for initialization here, just `let keys;` is enough.